app/views/user/_owner_profile.html.slim

```diff
@@ -1,35 +1,35 @@
--user_name = @user.display_name.presence || @user.login
+-user_name = user.display_name.presence || user.login
```

Comment:
This line and similar ones are just a change from using `@` instance variables to local ones in order to reduce unexpected reach. You'll see them being passed explicitly to this partial in  `app/views/user/profile.html.slim`.